database/index.js

```diff
@@ -2,8 +2,8 @@ const mongoose = require('mongoose');
 mongoose.connect('mongodb://localhost/shirts', { useNewUrlParser: true });
```

Comment:
nit: Not a necessary change, but just something to think about, instead of having a shirt table with a shirt schema, should we generalize it and just say product?